### PR TITLE
Minor Charter Corrections

### PR DESCRIPTION
**Pull request by LorbusChris**
Fix punctuation and semantics in a few places.
Thanks @jamescassell @cgruver @pecameron

WIP. A Call for Agreement will be made for this.

### CODE REVIEW

File: CHARTER.md
Comment (jamescassell) [inline CHARTER.md]: There should be a space after the comma.

File: CHARTER.md
Comment (jamescassell) [inline CHARTER.md]: Singular working group at the end.

**Review by jamescassell — COMMENTED** [review]
Two typos, otherwise looks good to me.

**Comment by DanyC97** [comment]
@LorbusChris happy to approve it once you remove the hold , just buzz me when is ready

**Comment by LorbusChris** [comment]
Let's approve this tentatively in the next meeting (per the process defined on the charter)

File: CHARTER.md
Comment (sgreene570) [inline CHARTER.md]: Should the first use of `SWG` be plural?

Comment (LorbusChris) [reply CHARTER.md]: yes, thanks!

**Review by Conan-Kudo — APPROVED** [review]

**Comment by LorbusChris** [comment]
approved in meeting
/hold cancel

**Pull request merged**
